src/app/components/QRCode/index.tsx

```diff
@@ -5,9 +5,12 @@ export type Props = {
   value: string;
   size?: number;
   className?: string;
+
+  // set the level to Q if there are overlays
```

Comment:
why? 

and why can't it always be "Q"?

Reply:
Always using Q will make the invoice QR codes unnecessarily more detailed, but it still works, right?

Reply:
Q will improve error correction (so we can add overlays covering up to 25% of the QR) at the price of decreased information density (meaning the QR codes "pixels" have to be smaller to encode the same information). 

While that isn't that much of a problem for lightning addresses (because they are usually quite short), for invoices that contain larger amount of data those QR codes can get "harder" to read. (meaning you have to aim your phone very precisely and have to wait longer for the reader to recognize the QR code)

The QR codes still work, but since we don't have overlays there (yet) I would prefer not to use the error correction and keep the QR codes pixels as big as possible for improved readability.

Reply:
ok, thanks for the explanation. makes sense.
can you add this to that comment there. the comment explains the what but not the why which we will not know in the future

Reply:
Done